pre_setup.py

```diff
@@ -0,0 +1,15 @@
+# For internal use. Please do not modify this file.
+
+def execute_cmd(cmd):
```

Comment:
Why do we need to leave this function here?

Reply:
I forgot to push the newer version here. Removing it.